web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -638,6 +642,12 @@
   })();
 
   window.liftJQuery = {
+    onEvent: function(elementOrId, eventName, fn) {
+      if (typeof elementOrId == 'string')
+        elementOrId = '#' + elementOrId;
+
```

Comment:
Thanks. I guess more than anything it just doesn't look right to me without braces as I tend to use them as a rule in JavaScript.